docs/tutorials/fundamentals/viewer.md

```diff
@@ -362,9 +362,35 @@ And to change the current position of the sliders use:
 viewer.dims.current_step = (3, 255, 255)
 ```
 The length of the `current_step` tuple corresponds to the number of dimensions. Note that in this example, the last two dimensions are *displayed* (don't have a slider) and thus changing the last two elements of the tuple will have no effect [until the axes order is changed](#roll-dimensions).
+The same information but in *world coordinates* (i.e., including
+scale and translate transformations) is accessible via `viewer.dims.point`.
 
-Lastly, `viewer.dims.point` contains the position in world coordinates (i.e., including
-scale and translate transformations).
+While by default napari will only show data located *exactly* at this position, it is possible to visualize data from a thicker dimensional slice by modifying `viewer.dims.margin_left` and `viewer.dims.margin_right`, or `viewer.dims.thickness` to change both symmetrically (or the corresponding `*_step` properties to work in "slider coordinates").
```

Comment:
```suggestion
By default napari will only show a slice of the data: that which is located *exactly* at this position. However, it is possible to visualize data from a thicker dimensional slice by modifying `viewer.dims.thickness`. This will use each layer's `projection_mode` to visualize the space around `viewer.dims.point`.  Alternately, you can set `viewer.dims.margin_left` and `viewer.dims.margin_right` to explicitly set the range of data around `viewer.dims.point` to be projected. Finally, you can use the corresponding `margin_left_step` and `margin_right_step` properties to work in "slider coordinates".
```
I found the original very dense and hard to understand. I think this is clearer? But maybe no longer correct 😅